docs/Anjay_integrations/Anjay_ST_integration/B-L475E-IOT01A.md

```diff
@@ -5,22 +5,33 @@ Integrate your B-L475E-IOT01A Discovery kit board.
 ## Prerequisites
 
 - The B-L475E-IOT01A board with a USB cable.
-- Installed **minicom** (for Linux) or RealTerm (for Windows) or other serial communication program.
+- Installed **minicom** (for Linux) or RealTerm or PuTTy (for Windows) or other serial communication program.
+- Installed **ST-Link** or **OpenOCD** debugger.
 - A user with access to the Coiote IoT Device Management platform and appropriate permissions.
 
-## Step 1: Getting Zephyr and Python dependencies
+## Prepare binaries
+### Use an already built binary
+
+To get the latest binaries, navigate to [Anjay-zephyr-client](https://github.com/AVSystem/Anjay-zephyr-client/releases).
+
+In this case we are going to use the demo_B-L475E-IOT01A1_merged.bin file which after being downloaded must be added to B-L475E-IOT01A board folder.
```

Comment:
`which after being downloaded must be added to B-L475E-IOT01A board folder.` same, what does it mean?